app/controllers/resources_controller.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+class ResourcesController < ApplicationController
+  # before_action :authorize_request
+
+  # POST /objects
+  def create
+    register_params
+    response = Dor::Services::Client.objects.register(params: register_params)
+    workflow_client.create_workflow_by_name(response[:pid], 'accessionWF')
+
+    render json: { druid: response[:pid] }, status: :created
+  rescue Dor::Services::Client::ConnectionFailed => e
+    render json: build_error('Unable to reach dor-services-app', e), status: :gateway_timeout
+  end
+
+  private
+
+  def workflow_client
```

Comment:
Place in an initializer like dor services client?

Reply:
I'm using this as a factory, not as a global initializer.   I think this is a better pattern.